armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -162,11 +162,7 @@ apt purge -y ntp network-manager
 apt update
 apt upgrade -y
 
-# development
-apt install -y  git tmux qrencode
-
-# system
-apt install -y  openssl net-tools fio libnss-mdns \
+apt install -y  git openssl net-tools fio libnss-mdns \
```

Comment:
(Optional) Not new in this PR, but I noticed we don't specify `--no-install-recommends` here, as we do elsewhere. Would it make sense to add it here, to only install packages we explicitly list?

Reply:
Good point, I'll tackle that in a next pull request with proper testing.